rom/dev/tools/test-fmc/src/main.rs

```diff
@@ -218,6 +233,13 @@ fn process_mailbox_commands() {
     process_mailbox_command(&mbox);
 }
 
+fn read_pcr31(mbox: &caliptra_registers::mbox::RegisterBlock<RealMmioMut>) {
+    let pcr_bank = unsafe { PcrBank::new(PvReg::new()) };
+    // Note: read_pcr returns PCR in big-endian; convert to little-endian before sending it to mailbox.
```

Comment:
It doesn't look like this comment is accurate? I don't see endian conversion here.

Reply:
Array4X12 changes the endianess.